packages/FormElement/src/FormElement.js

```diff
@@ -87,14 +87,18 @@ function FormElement(props) {
 
   function renderFooter() {
     if (hasError) {
-      return React.cloneElement(extractedChildren["FormElement.Error"], {
-        ariaErrorId,
-      });
+      return (
+        <div aria-live="assertive">
```

Comment:
@sunminsarahlee I am using the aria-live attribute of "assertive" here has this is an error being announced, and as the visual representation is an exclamation mark, is red and is designed to draw attention, i felt the screen reader should also reflect that importance. Once the error has been read out, the screen reader (on mac) defaults to letting the listener know they are on a button, so the listener won't lose context.

Let me know what you think.